bundles/org.openhab.binding.huesync/src/main/java/org/openhab/binding/huesync/internal/HueSyncConstants.java

```diff
@@ -0,0 +1,104 @@
+    }
+
+    public static class CHANNELS {
+        public static class DEVICE {
+            public static class INFORMATION {
+                public static final String FIRMWARE = "device-firmware#firmware";
+                public static final String FIRMWARE_AVAILABLE = "device-firmware#available-firmware";
+            }
+        }
+
+        public static class COMMANDS {
+            public static final String MODE = "device-commands#mode";
+            public static final String SYNC = "device-commands#syncActive";
+            public static final String HDMI = "device-commands#hdmiActive";
+            public static final String SOURCE = "device-commands#hdmiSource";
+            public static final String BRIGHTNESS = "device-commands#brightness";
+        }
+
+        public static class HDMI {
+            public static class IN_1 {
+                public static final String NAME = "device-hdmi-in-1#name";
+                public static final String TYPE = "device-hdmi-in-1#type";
+                public static final String STATUS = "device-hdmi-in-1#status";
+                public static final String MODE = "device-hdmi-in-1#mode";
+            }
+
+            public static class IN_2 {
+                public static final String NAME = "device-hdmi-in-2#name";
+                public static final String TYPE = "device-hdmi-in-2#type";
+                public static final String STATUS = "device-hdmi-in-2#status";
+                public static final String MODE = "device-hdmi-in-2#mode";
+            }
+
+            public static class IN_3 {
+                public static final String NAME = "device-hdmi-in-3#name";
+                public static final String TYPE = "device-hdmi-in-3#type";
+                public static final String STATUS = "device-hdmi-in-3#status";
+                public static final String MODE = "device-hdmi-in-3#mode";
+            }
+
+            public static class IN_4 {
+                public static final String NAME = "device-hdmi-in-4#name";
+                public static final String TYPE = "device-hdmi-in-4#type";
+                public static final String STATUS = "device-hdmi-in-4#status";
+                public static final String MODE = "device-hdmi-in-4#mode";
+            }
+
+            public static class OUT {
+                public static final String NAME = "device-hdmi-out#name";
+                public static final String TYPE = "device-hdmi-out#type";
+                public static final String STATUS = "device-hdmi-out#status";
+                public static final String MODE = "device-hdmi-out#mode";
+            }
+        }
+    }
+
+    public static final String APPLICATION_NAME = "openHAB";
+
+    /** Minimal API Version required. Only apiLevel >= 7 is supported. */
+    public static final Integer MINIMAL_API_VERSION = 7;
+
+    public static final String BINDING_ID = "huesync";
+    public static final String THING_TYPE_ID = "huesyncthing";
```

Comment:
Resolved by 45babc6a887a804eb4b0a8c5a6d872fd9b30177d

Reply:
Now it changed from `huesyncthing` to `huesyncbox` and that is not what i meant.
1. There is no need to name the binding id again in the thing id. So the `huesync` prefix has no meaning here. It is rather confusing.
2. I would just name it `device` or `box`
3. If you schoose something else it should be `lower-case-hyphen`

Reply:
👍️ makes sense to avoid redundancy if possible. Applied the change in:
- 13eedb09f6ed207711f9a07b63600098522451f8

Please x-check and mark as resolved if you agree with the change.